### PR TITLE
fix: handle timeout

### PR DESCRIPTION
**Pull request by gideonairex**

### CODE REVIEW

**Comment by gideonairex** [comment]
@yamii @openam @fakewaffle heres the PR for timeouts instead of deadletter.

**Comment by gideonairex** [comment]
@openam updated to 40 seconds.

File: lib/req-res/status.js [hunk 1]
```diff
@@ -4,59 +4,84 @@ module.exports = function ( options ) {
...
-				'status' : 'success',
-				'data'   : data
+	var timeout = setTimeout( function () {
+		responded = true;
+		respond( {
+			'status' : 'fail',
+			'data'   : 'Request timeout'
+		} );
+		setImmediate( function () {
+			log.silly( 'success', {
```
Comment (openam) [inline lib/req-res/status.js]: Why is this `success`?

Comment (gideonairex) [inline lib/req-res/status.js]: oh no. will fix this.

Comment (gideonairex) [inline lib/req-res/status.js]: What would be right? fail? error?

Comment (gideonairex) [inline lib/req-res/status.js]: and timeout?

Comment (gideonairex) [inline lib/req-res/status.js]: ```log.fail( 'timeout' )``` ?

Comment (gideonairex) [inline lib/req-res/status.js]: @openam addressed it here [54](https://github.com/School-Improvement-Network/lapin/pull/54)

**Pull request merged**